### PR TITLE
perf(ingest): enable orjson experiment for ingest

### PR DESCRIPTION
**Pull request by anonrig**
Experimenting on improving the performance of JSON parsing with @fpacifici  

Ref: https://github.com/getsentry/team-ingest/issues/390

### CODE REVIEW

**Review by fpacifici — COMMENTED** [review]
Looks good to me. 
@jan-auer  who should review from ingest ?

**Review by jan-auer — APPROVED** [review]

**Comment by fpacifici** [comment]
Please get a review from the ingest team as well.

**Review by fpacifici — APPROVED** [review]

**Pull request merged**
